### ISSUE TITLE
Voting Proposals Cards Need More Information

### ISSUE THREAD

**Issue by FMA126** [iearn-finance/ygov-finance#3]
Example of the available information for each proposal:
```
====== PROPOSAL #0 ======
- Minting more $YFI - 

FOR     : Allows weekly distribution of YFI. A second proposal will be submitted to decide how much would be printed weekly.
AGAINST : No more YFI tokens will be distributed. Global supply is locked at 30000 YFI permanently.

https://twitter.com/iearnfinance/status/1284454945614442499

Proposed by         : 0x473afAb58B2C5D4DbC5FAD5D236f6658AD84E83b
Total for votes     : 3435983.4406505744 (41.15%)
Total against votes : 4913869.508808158 (58.84%)
Start block         : 10490942
End block           : 10508222

====== PROPOSAL #1 ======
- Burn $YFI for fees - 

yearn.finance is currently earning $60k/week in protocol fees, which can grow in the future with yearn.finance ecosystem growth and new products.

FOR     : Will continue to with the current burn model.
AGAINST : Rewards will be claimed via a staking model instead.

https://twitter.com/iearnfinance/status/1284531647078178816

Proposed by         : 0x473afAb58B2C5D4DbC5FAD5D236f6658AD84E83b
Total for votes     : 502445.8576 (11.48%)
Total against votes : 3870866.548141156 (88.51%)
Start block         : 10490942
End block           : 10508222
```

**Comment by antonnell** [comment]
This needs to be manually coded every time a new vote is created, so I'm probably not going to do it.
Community is welcome to submit a PR every time a new vote is put out.

**Comment by antonnell** [comment]
Link to the YIP is now in the vote card.
Closed by #10
